src/ventus/wm/modes/expose.js

```diff
@@ -4,30 +4,72 @@
  * https://github.com/rlamana
  */
 define([
+	'$',
 	'underscore',
 	'ventus/core/promise'
-], function(_, Promise) {
+], function($, _, Promise) {
 	'use strict';
 
 	var ExposeMode = {
 		// Launch when plugin is registered
 		register: function() {
 			var self = this;
+			var listener;
+			var listeners = this.exposeListeners;
 
-			console.log('Expose mode registered.');
-
-			this.el.on('contextmenu', _.throttle(function() {
-				// Right click sets expose mode
+			var setMode = function (){
 				if (self.mode !== 'expose') {
-					if(self.windows.length > 0) {
+					if (self.windows.length > 0) {
 						self.mode = 'expose';
 					}
-				} else if(self.mode === 'expose') {
+				} else {
 					self.mode = 'default';
 				}
+			};
+
+			var keypressCallback = function (keyCode) {
+				return {
+					selector: $(document),
+					on: 'keypress',
+					callback: function (event) {
+						if (event.keyCode === keyCode) {
+							setMode();
+						}
+					}
+				};
+			};
+
+			var validListeners = {
+				a: keypressCallback(97),
```

Comment:
also not very important, but technically that function is not a callback nor it returns a callback exactly. I would rename it to something more descriptive like ```configKeyListener```.